<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageRoutesWindow</class>
 <widget class="QWidget" name="ManageRoutesWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>297</width>
    <height>184</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>297</width>
    <height>184</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>297</width>
    <height>184</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Управление рейсами</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/new/prefix1/icons1/samolet.jpg</normaloff>:/new/prefix1/icons1/samolet.jpg</iconset>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>20</y>
     <width>254</width>
     <height>155</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>ID:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSpinBox" name="idEdit">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>9999</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>П. отправления:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QLineEdit" name="depEdit">
      <property name="maxLength">
       <number>40</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>П. назначения:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QLineEdit" name="desEdit">
      <property name="maxLength">
       <number>40</number>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Марка самолёта:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="2">
     <widget class="QLineEdit" name="markEdit">
      <property name="maxLength">
       <number>8</number>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Количество мест:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QSpinBox" name="seatsEdit">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>200</number>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="closeButton">
      <property name="text">
       <string>Закрыть</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="deleteButton">
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="createButton">
      <property name="text">
       <string>Создать</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QCheckBox" name="checkBox">
      <property name="text">
       <string>Удаление</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>ManageRoutesWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>148</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deleteButton</sender>
   <signal>clicked()</signal>
   <receiver>ManageRoutesWindow</receiver>
   <slot>deleteRoute()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>148</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>createButton</sender>
   <signal>clicked()</signal>
   <receiver>ManageRoutesWindow</receiver>
   <slot>createRoute()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>237</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>148</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>ManageRoutesWindow</receiver>
   <slot>reactToCheck()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>148</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>deleteRoute()</slot>
  <slot>createRoute()</slot>
  <slot>reactToCheck()</slot>
 </slots>
</ui>
